app/models/file_set.rb

```diff
@@ -4,4 +4,8 @@ class FileSet < Valhalla::Resource
   attribute :id, Valkyrie::Types::ID.optional
   attribute :title, Valkyrie::Types::Set
   attribute :member_ids, Valkyrie::Types::Array
+
+  def thumbnail_id
+    @thumbnail_id ||= Valkyrie.config.metadata_adapter.query_service.find_members(resource: self).find { |x| x.use.include?(Valkyrie::Vocab::PCDMUse.ServiceFile) }.try(:id)
```

Comment:
I mean the "/full/!200,150/0/default.jpg" part that gets added at https://github.com/pulibrary/figgy/pull/42/files#diff-55f651679ae90d845871424d7d4e9bd3R180

So this method would return the entire thumbnail URL